solutions/src/main/java/ru/sberbank/school/task03/FxRequestConversionService.java

```diff
@@ -0,0 +1,15 @@
+package ru.sberbank.school.task03;
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+
+public interface FxRequestConversionService {
+
+    public Symbol getSymbol (String symbol);
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1497116708>*